packages/react/src/select/scroll-arrow/SelectScrollArrow.tsx

```diff
@@ -67,67 +70,121 @@ const SelectScrollArrow = React.forwardRef(function SelectScrollArrow(
           style: {
             position: 'absolute',
           },
-          onMouseEnter() {
-            if (!alignItemToTrigger) {
+          onMouseMove(event) {
+            if (
+              (event.movementX === 0 && event.movementY === 0) ||
```

Comment:
I'm curious in what cases `onMouseMove` fires with `event.movementX === 0 && event.movementY === 0`?

Reply:
<img width="58" alt="Screenshot 2025-03-19 at 7 00 47 pm" src="https://github.com/user-attachments/assets/4ccb913a-16f9-4293-bc64-efc2654bc5ac" />

When moving slowly it can happen frequently